environments/argocd-app-templates/appsetup-int.yaml

```diff
@@ -36,3 +36,45 @@ spec:
         - name: helm_args
           value: '-f values.yaml -f ../../environments/helm-values/values-int.yaml'
   project: project-portal
+data:
```

Comment:
the configmap should be an own file in the templates dir of the helm chart

Reply:
adjusted :)
 - thanks for the help